scripts/build_packages.py

```
@@ -0,0 +1,330 @@
+    for node in ast.walk(tree):
+        if isinstance(node, ast.Call) and getattr(node.func, 'id', '') == 'setup':
+            for keyword in node.keywords:
+                if (keyword.arg == 'install_requires' and
+                    isinstance(keyword.value, (ast.List, ast.Tuple))):
+                    all_deps.extend([
+                        elt.value for elt in keyword.value.elts
+                        if isinstance(elt, ast.Constant) and isinstance(elt.value, str)
+                    ])
+                elif (keyword.arg == 'extras_require' and
+                      isinstance(keyword.value, ast.Dict)):
+                    for key_node, val_node in zip(keyword.value.keys, keyword.value.values):
```

Ruff (0.12.2)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)